include/cetl/unbounded_variant.hpp

```diff
@@ -588,7 +617,7 @@ struct base_access : base_storage<Pmr, Footprint, Alignment>
         }
         CETL_DEBUG_ASSERT(nullptr != value_const_converter_, "Non-empty storage is expected to have value converter.");
 
-        return value_converter_(base::get_raw_storage(), type_id_value<ValueType>);
+        return value_mut_converter_(base::get_raw_storage(), cetl::type_id_value<ValueType>);
```

Comment:
Renamed to emphasise that there are two distinct converts: mutable and constant.